### PR TITLE
Linkedin Wish Sender

### LINKED ISSUE #1020: Send automated messages on LinkedIn
# Description

LinkedIn shares a notification if one of our first-degree connections is having a birthday, or work anniversary or has landed a new role. It would be a great job if we can automate, the whole workflow of sending them best regards by relying on the automated reply given by LinkedIn. You can make use of a lightweight database (JSON) to track the messages that have been sent already so that the script does not ping them back. You can also add a Cron Job so that the script can be run at scheduled times.

## Language

- [X] Go
- [X] Javascript
- [X] Python
- [X] BASH

### PR DESCRIPTION
**Pull request by Sukriti-sood**
# Description

Added Script that Sends the Wishes by Message to the first Connections on Linkedin on their Birthdays or any Other Good News Like Start New Position, Work Anniversary etc.

Fixes #1020 

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Python/Linkedin_Wishes_Sender/credentials.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "username": "Enter your Email Address",
+    "password": "Enter your Password"
+}
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/credentials.json:4]: Add a new line at the EOF

File: Python/Linkedin_Wishes_Sender/Readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+This Script Sends the Wishes by Message to the first Connections on Linkedin on their Birthdays or any Other Good News Like Start New Position, Work Anniversary etc.
+
+
+
+## Requirements
+
+- Linkedin Profile
+- Python Setup on System
+- Chromium
+- Chrome
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/Readme.md:12]: Redundant requirements; Everyone using the Script should have these

File: Python/Linkedin_Wishes_Sender/Readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Install Selenium
+- Install bs4 (```pip3 install bs4```)
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/Readme.md:14]: Put a `requirements.txt` and put all of the dependencies there

File: Python/Linkedin_Wishes_Sender/Readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- Replace usernmae and password in credentials.json with your gmail and password
+
+## How to Run
+
+```python3 script.py```
+
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/Readme.md:20]: Can you add a GIF showing the script in action if possible?

File: Python/Linkedin_Wishes_Sender/Readme.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Linkedin Wishes Sender
+
+This Script Sends the Wishes by Message to the first Connections on Linkedin on their Birthdays or any Other Good News Like Start New Position, Work Anniversary etc.
+
+
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/Readme.md:5]: Remove extra lines here

File: Python/Linkedin_Wishes_Sender/script.py [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+def main():
+    browser = webdriver.Chrome()
+    browser.maximize_window()
+    wish(browser)
+
+
+
+if __name__ == "__main__":
+    main()
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/script.py:116]: Add a new line at the EOF

File: Python/Linkedin_Wishes_Sender/script.py [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+        browser.close()        
+
+
+
+def main():
+    browser = webdriver.Chrome()
+    browser.maximize_window()
+    wish(browser)
+
+
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/script.py:113]: Remove extra lines of code wherever possible

File: Python/Linkedin_Wishes_Sender/script.py [hunk 8]
```diff
@@ -0,0 +1,116 @@
+# All Imports
+from selenium import webdriver
+import datetime
+from bs4 import BeautifulSoup
+import json
+import time
+
+
```
Comment (HarshCasper) [inline Python/Linkedin_Wishes_Sender/script.py:8]: Remove extra lines of code wherever possible

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
@Sukriti-sood Are you participating in some OSS program apart from PSoC '21 ?

**Comment by Sukriti-sood** [comment]
> @Sukriti-sood Are you participating in some OSS program apart from PSoC '21 ?

yes ,LGM-SOC

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**